### PR TITLE
Update ru.coffee

### PR DESCRIPTION
**Pull request by rybinavasd**

### CODE REVIEW

**Comment by mrfinch** [comment]
@rybinavasd can u sign cla here: https://codecombat.com/cla so that i can review your code, thanks.

**Comment by rybinavasd** [comment]
> @rybinavasd can u sign cla here: https://codecombat.com/cla so that i can review your code, thanks.

done

File: app/locale/ru.coffee [hunk 1]
```diff
@@ -2887,7 +2887,7 @@ module.exports = nativeDescription: "русский", englishDescription: "Russi
     third_party_description: "CodeCombat использует следующие сторонние сервисы (среди прочих):"
     cookies_message: "CodeCombat использует постоянные и сеансовые куки."
     cookies_deny: "Отказаться от использования сеансовых куки"
-#    cookies_allow: "Allow cookies"
+    cookies_allow: "Разрешить файлы cookie" # {change}
```
Comment (mrfinch) [inline app/locale/ru.coffee:2890]: remove # {change}

Comment (rybinavasd) [reply app/locale/ru.coffee]: > remove # {change}

removed

**Review by mrfinch — APPROVED** [review]

**Pull request merged**